sys-devel/clang-common/clang-common-19.0.0.9999.ebuild

```diff
@@ -12,9 +12,10 @@ LICENSE="Apache-2.0-with-LLVM-exceptions UoI-NCSA"
 SLOT="0"
 IUSE="
 	default-compiler-rt default-libcxx default-lld
-	bootstrap-prefix cet hardened llvm-libunwind
+	bootstrap-prefix cet hardened llvm-libunwind polly
 "
 
+DEPEND=" polly? ( ~sys-devel/polly-${PV} ) "
```

Comment:
`DEPEND` is for build-time dependencies, and this ebuild definitely does not need polly at build-time. It should go into `PDEPEND`, or `RDEPEND`. We've been using `PDEPEND` so far to avoid circular dependencies but perhaps only `clang` should actually be a PDEP.

@thesamesam, WDYT?

Reply:
The issue here is that polly needs to be installed and present **before** clang-common or, otherwise compile problems might appear. I.e. a LLVM major update using polly flags in CXXFLAGS will fail if clang-common is installed before polly.

Reply:
I'd suggest the same for libc++ since I have had problems with this being enabled before libc++ is installed/updated, specially after symbol changes from recompiles.

Reply:
That's what `RDEPEND` is for. `DEPEND` will be only used at build time, and can be unmerged afterwards. And it won't be installed at all when using binary packages.